vademecum.tex

```diff
@@ -325,6 +325,17 @@ \section{Masters in HPC and SC}
 \end{itemize}
 Starting from from the academic year 2020/2021, SISSA has introduced an important novelty regarding the admission to the MHPC: SISSA waves the fee of the Master to its PhD students who are enrolled on one of its PhD courses on the date of the official start of MHPC classes, and to former students who have obtained their doctorate no more than 12 months before the start date of the lessons and that are currently unemployed, for a maximum of \underline{six} exemptions. The exemption is granted to the first three PhD students and to the first three former PhD students that are admitted to the Master, according to the ranking of the admission. Should the number of admitted students or former students be less than three, the exemptions not enjoyed by one of the two categories will be transferred to the other. Beneficiaries of the exemption are required to complete the training course, under penalty of paying € 1.000,00 to SISSA, as a contribution to the training costs. Beware that, if the beneficiary of an exemption is awarded a scholarship that also includes a contribution for the fees (for example, a contribution offered by other research institutions or private companies), the benefit of the exemption is withdrawn and the student is required to pay the balance of the entire fee for participation in MHPC. If this disclaimer has ignited your interest, have a look at how the application process works at \href{https://www.mhpc.it/how-apply}{this link}!
 
+\section{Temporary unemployment benefit}
+
+If you do not have a Gestiona Separata completed -- it is the first step to do.
+\begin{itemize}
+   \item The first way is to go directly to INPS office Trieste ( Via Sant'Anastasio, 5,), nearby busstop 38. Probably I didn't even fix an appointment- just came and had a talk. In principle, they should give you INPS code. But I had some problems, so I have obtained a SPID code (see (2.) )
```

Comment:
The first way is to go directly to the INPS office in Trieste (Via Sant'Anastasio, 5), near the bus 38 stop. Most probably, you didn't even need to fix an appointment: you can just pop over there and have a chat. In principle, they should give you an INPS code or a SPID code, see the next point.